### ISSUE TITLE
Does NOT work with Bootstrap 3

### ISSUE THREAD

**Issue by jaycbrf4** [gudh/ihover#11]
Bootstrap is the most popular HTML, CSS, and JS framework for developing responsive, mobile first projects on the web.

Your effects are not responsive or mobile first.
![screen shot 2015-02-23 at 10 39 27 am](https://cloud.githubusercontent.com/assets/757717/6330672/47045a34-bb48-11e4-9fa1-98314e562b91.png)

If this effect did "Work well with Bootstrap 3 (Bootstrap is not necessary though)." as stated on your demo page. Then the images would not overlap, they would scale down.

Please fix or remove "Work well with Bootstrap 3 (Bootstrap is not necessary though)."

**Comment by BowuDev** [comment]
I agree __100%__ I've seen it _overrides_ the bootstraps themes (_ui_) D= please fix it!

**Comment by gudh** [comment]
@NavyCo thanks for your feedback!

I am gonna do an update recently to make it work with the coming Bootstrap 4

**Comment by BowuDev** [comment]
OK that'd be great =D, I'm gonna have to use an alternative for now, if I can find one as greatly done as yours >->

**Comment by gudh** [comment]
@NavyCo By the way, could give me more details on that?

Because I used Bootstrap 3 in the demo, and it's fine
